io.catenax.shared.standard_codes/1.0.0/standard_codes.ttl

```diff
@@ -0,0 +1,71 @@
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.shared.standard_codes:1.0.0#> .
+@prefix samm-u: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+
+:country a samm:Property ;
+   samm:preferredName "Country"@en ;
+   samm:description "Alphabetic three-character (alpha-3) code representing a country name"@en ;
+   samm:see <https://www.iso.org/iso-3166-country-codes.html> ;
+   samm:characteristic :CountryAlpha3Trait ;
+   samm:exampleValue "DEU" .
+
+:CountryAlpha3Trait a samm-c:Trait ;
+   samm:preferredName "Country Alpha 3 Trait "@en ;
+   samm:description "3 letter country code"@en ;
+   samm-c:baseCharacteristic :Text ;

```

Comment:
There is a predefined Characteristic for Text, please use samm-c:Text and remove : Text